hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestSyncTimeRangeTracker.java

```diff
@@ -34,7 +36,7 @@
   public static final HBaseClassTestRule CLASS_RULE =
       HBaseClassTestRule.forClass(TestSyncTimeRangeTracker.class);
 
-  private static final int NUM_KEYS = 10000000;
+  private static final int NUM_KEYS = 1000000;
```

Comment:
So this is the actual fix right?

Reply:
This is kind of separate. I noticed this huge array took a lot of RAM and it did not seem necessary.

I also noticed that the array is so large that the threads working on it don't overlap much as is.